openfl-tutorials/interactive_api/Tensorflow_MNIST/envoy_two_folder/mnist_shard_descriptor.py

```diff
@@ -0,0 +1,83 @@
+# Copyright (C) 2020-2021 Intel Corporation
```

Comment:
No need to repeat code. I suggest leaving only one SD implementation and several envoy configs, pointing to it

Reply:
I created two folders (with the same shard code) specifically for easy use, to use the two collaborators, we just need to copy the folder to another node.